packages/site/src/content/docs/blog/introducing-flint.md

```diff
@@ -0,0 +1,160 @@
+
+:::tip
+For more information on Flint's plugins, see:
+
+- [Glossary](/glossary): for the common linting terms as used by Flint, and compared to other linters
+- [Rules](/rules): to see the provided plugins and rules to be (or not to be) implemented in them
+
+:::
+
+### Tooling Coordination
+
+Modern web projects are able to use a plethora of tools: formatters, a variety of linters, type-checkers, and more.
+Each of those tools often ends up being either its own workflow in CI and/or a step in a larger CI job.
+Repositories have to choose: how much tooling power do they want, at the cost of CI time and configuration complexity?
+
+Flint aims to reduce complexity and user pain in three ways:
+
+- Flint's APIs allow for project-aware lint rules that can take on complex tasks like import cycle analysis and unused code detection
+- Flint coordinates a formatter ([Prettier](https://prettier.io)) out-of-the-box, running on all files being linted
+- Flint allows languages to surface "diagnostics" such as TypeScript's type errors as part of linting as well
+
+Flint's hypothesis is that providing one centralized place to coordinate all those tasks, users' repositories will become simpler and benefit from faster and less complex CI workflows.
+
+## The Flint Project
+
+The goal of the Flint project is to _test its hypothesis_.
+Flint is run by a set of independent volunteer maintainers as a fully open source project.
+We are not associated with any one company, and not pressured by funding to achieve any level of adoption in industry.
+
+The best case scenario would be that all of Flint's hypotheses would be shown to be correct.
+As in, Flint's choices do lead to a linter that is fast by default, straightforward to configure, and useful for users.
+In that case we'll transition Flint to a full open source project to stand alongside other web linters.
+
+Another success case would be that some or all of the hypothesis are proven wrong.
+In that case we'll feel more confident that traditional linters such as ESLint and/or native speed linters such as Biome and Oxlint are the way to go.
+
+### Getting Involved
+
+If you're interested in helping build out a new experimental linter, we'd love to have you join us.
+At the very least, see [About](/about) for how to get started using Flint as a user.
+Just trying out the project and telling us about your experience on the [Flint Discord](https://discord.gg/cFK3RAUDhy).
+
+The [GitHub issue tracker](https://github.com/JoshuaKGoldberg/flint/issues) is where you can find our list of upcoming work.
+See our [Contributing guide](https://github.com/JoshuaKGoldberg/flint/blob/main/.github/CONTRIBUTING.md) for how to find issues that you can get started on.
+Feel free to ask for help if you're new and unsure.
+We're happy to assist you.
+
+### Supporting Flint Financially
+
+Flint can receive donations on its [Open Collective](https://opencollective.com/flint.fyi).
```

Comment:
I submitted the application for this today. It's pending. The last application I sent to OC was responded to within a business day and finalized within a couple more business days. 🚀